src/components/audio_instructions_web/_styles.scss

```diff
@@ -0,0 +1,4 @@
+.ffwdme-components-audio-instructions {
```

Comment:
Could you modify the namespace to `ffwdme-components-audio-instructions-web`?

Reply:
done